change/@uifabric-example-app-base-2019-07-10-17-15-36-monaco-editor.json

```diff
@@ -0,0 +1,8 @@
+{
+  "comment": "Website changes to work with monaco editor state management",
```

Comment:
>monaco [](start = 43, length = 6)

(nit) capitalize

Reply:
Came from commit message, will capitalize in future commits

---
In reply to: [302627938](https://github.com/OfficeDev/office-ui-fabric-react/pull/9769#discussion_r302627938) [](ancestors = 302627938)